Source/Quick Start Documentation/FPGA Addon Quick Start Guide.md

```diff
@@ -144,7 +144,7 @@ An addon configuration can be imported with the following dialog box:
 
 ---
 
-After the FPGA Addon has been added, set the **Bitfile path** to ​​​the desired location.​​ You must select the **RIO Device**. Using the search (magnifier glass) will display available FPGAs on the specified Target or Controller.
+After the FPGA Addon has been added, set the **Bitfile path** to ​​​the desired location.​​ You must select the **RIO Device**. Using the search (magnifier glass) will display available FPGAs on the specified Target or Controller. An alias can be entered manually as well.
```

Comment:
You can either use the magnifying glass, or enter manually an alias. So it can either be step 2, or step 3, as step 3 will cancel what you did in step 2.

Reply:
@phutanu Here is another version:

"After adding the FPGA Addon, configure the add-on settings in System Explorer.
  1. Set the Bitfile path.
  2. Select the RIO Device using either of the following options: 
    - Click the magnifying glass icon. This will display the available FPGAs on the specified target or controller.
    - Enter an alias manually."

Reply:
Great, thanks @WillCawley! :)